lib/react.dart

```diff
@@ -318,72 +465,433 @@ class SyntheticWheelEvent extends SyntheticEvent {
 
 }
 
-/**
- * client side rendering
- */
+/// Client side rendering.
 @deprecated
```

Comment:
for deprecated methods, it would be nice to include a comment about the replacement API, like you did for `getDOMNode`